Gemfile

```diff
@@ -33,9 +33,6 @@ gem 'jbuilder', '~> 2.5'
 group :development, :test do
   # Call 'byebug' anywhere in the code to stop execution and get a debugger console
   gem 'byebug', platform: :mri
-  gem 'rspec-rails'
-  gem 'factory_girl_rails'
-  gem 'database_cleaner'
```

Comment:
We should keep at least rspec-rails and factory_girl_rails in the development group, so when we use generators for models, controllers, views and helpers, our spec files and factories are automatically created as well.

The database_cleaner gem could be left out the development group, but wouldn't hurt to have it into too. And as we didn't have a test group into the Gemfile, I opted to leave on the development, test group